addons/website/static/src/js/menu/edit.js

```diff
@@ -298,6 +298,16 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
                 $savable.not('.o_dirty').each(function () {
                     const $el = $(this);
                     if (!$el.closest('[data-oe-readonly]').length) {
+                        // Displaying the link popover after clicking on a link
+                        // should not set it as o_dirty.
+                        const hasTip = $el.popover('getTipElement');
+                        if (hasTip) {
+                            const tipAttributeMutations = ['data-original-title', 'title', 'aria-describedby'];
+                            const isTipRelated = record.type === 'attributes' && tipAttributeMutations.includes(record.attributeName);
+                            if (isTipRelated) {
+                                return;
+                            }
+                        }
```

Comment:
I guess it is ok but should not it be about disabling the observer for the duration of the link popover opening? Seems like we went that way more often than processing the mutation (not sure this is better). What do you think?

Also, typo in commit message (I see at least a double space)

Reply:
After taking another look I found out that these mutations were not related to the LinkPopoverWidget tooltips, which are targeting the popover itself, not the link being edited, but from the "Double-click to edit" tooltip... 

In the end, it seems better to deactivate the observer when invoking these tooltips than filtering globally on the tooltips-related mutations. 
The global approach had the advantage that we would not need to deactivate the observer again when introducing new tooltips at other places, but in the current state, a snippet option that would change one of these three attributes on a link and nothing else would not be saved...

So I'm changing that to deactivating the observer for the "Double-click to edit" tooltip !